content/rs/administering/designing-production/networking/port-configurations.md

```diff
@@ -29,6 +29,8 @@ update your firewall with the port for that new database endpoint.
 | 8001 | Used by your application to access the RS [Discovery Service]({{< relref "/rs/concepts/data-access/discovery-service.md" >}}) | TCP |
 | 8443 | Used for secure (https) access to the management web UI | TCP |
 | 8444, 9080 | Used for nginx \<-\>cnm_http/cm communications on the same host only. Ports are bound to loopback adapter. | TCP |
+| 9081 | Used for crdb_coordinator | TCP |
```

Comment:
"Used for CRDB management"

Reply:
@RoeyPrat - can you approve it please? Maybe we can use a better explanation that does not use the internal "unknown" phrase of crdb coordinator?

Reply:
yes, 9081 is used by crdb_coordinator. @yossigo @guybe7 @oranagra  FYI

yes 8070, 8071 are used by metrics_exporter. @danni-m FYI

Reply:
Note that it's a loopback port, used to communicate with nginx only and not exposed outside the node.

Reply:
I think port that are used should be specified, regardless of whether they are exposed. 
Perhaps it is a good idea to specify which are exported in the table, or put the internal ones in a separate table.

Reply:
@yossigo Which port is loopback?

Reply:
@RoeyPrat If loopback traffic doesn't pass a firewall between nodes then why list it?

Reply:
@bmansheim port 9081, used by crdb_coordinator.  Loopback local ports don't need to be punched through firewalls, but they may conflict other processes running on the same node so I see value in documenting them.

Reply:
I agree with yossi on this.
install.sh checks if these ports are available, and fails installation if they are not.
It also points user to docs for explanation.